src/PrestaShopBundle/Resources/views/Admin/Improve/Design/MailTheme/index.html.twig

```diff
@@ -38,6 +38,12 @@
     })
   }}
 
+  {{
```

Comment:
Just because two last lines disturb me
```php
  {{
    include(
      '@PrestaShop/Admin/Improve/Design/MailTheme/Blocks/translate_mails_body_form.html.twig', 
      {
        'generateMailsForm': generateMailsForm
      }
    )
  }}
```

Reply:
Done, not exactly as you suggested but I used the same indentation as other includes in the file, which look better